docs/DEBUGGING.md

```diff
@@ -88,187 +88,187 @@ $ cat /persist/agentdebug/zedbox/sigusr1 | sed 's/\\n/\'$'\n''/g' | sed 's/\\t/\
 {"file":"/pillar/agentlog/agentlog.go:71","func":"github.com/lf-edge/eve/pkg/pillar/agentlog.handleSignals","level":"warning","msg":"SIGUSR1 triggered stack traces:
 goroutine 10 [running]:
 github.com/lf-edge/eve/pkg/pillar/agentlog.getStacks(0xc000080101, 0xc000000004, 0x1465007)
-	/pillar/agentlog/agentlog.go:218 +0x78
+    /pillar/agentlog/agentlog.go:218 +0x78
```

Comment:
Maybe the langauge tag should be text instead of sh above. 
In anyn case, it doesn't make sense replacing the output of a command example just because blanks doesn't like it.